triaging.rst

```diff
@@ -43,11 +43,32 @@ or a veteran core developer, they're actively choosing to voluntarily donate the
 time towards the improvement of Python. As is the case with any member of
 the Python Software Foundation, always follow the `PSF Code of Conduct`_.
 
-.. note::
+Becoming a member of Python Triage team
+---------------------------------------
+
+Any Python core developers are welcome to invite a Python contributor to the
+Python Triage team. Do note that the responsibilities of a Python Triage member
+is more elevated than a bug triager in bpo. For example, the Python Triage member
+has access to more repositories than just CPython. Triages will be responsible
+to handle not just issues, but also pull requests, and even commanding backports.
+
+Any existing triagers in b.p.o can transition to the Python Triage role.
+They can request this to any core developer, and the core developer
```

Comment:
```suggestion
They can request this for any core developer, and the core developer
```

Reply:
"to" sounds right to me, no? 🤔

Reply:
Agreed, I think "to" is okay in this context.

Reply:
Ah yes, "to" is correct here, I got it mixed up. Thanks!